Uppsala/UppEt9.xml

```diff
@@ -0,0 +1,323 @@
+                                <extent>
+                                    <measure unit="leaf">174</measure>
+                                    <note>Two endleaves at the end.</note>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>235</height>
+                                        <width>210</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="17 18">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic" corresp="ms_i1">
+                                <desc>Good, regular hand (<foreign xml:lang="de">[…] guten gleichmäßigen Hand</foreign>, 
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">47</citedRange></bibl>).
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">45</citedRange></bibl>
+                                    dates it to the 18th–19th century, whereas 
+                                    <bibl><ptr target="bm:Zettersteen1899Upsala"/><citedRange unit="page">518</citedRange></bibl>
+                                    suggests that it might date from the 19th century.</desc>
+                                <date notBefore="1700" notAfter="1899">18th–19th century.</date>
+                            </handNote>
+                            
+                            <handNote xml:id="h2" script="Ethiopic" corresp="#a1">
+                                <desc>Clear hand (<foreign xml:lang="de">deutlich</foreign>), 
+                                    smaller and later than <ref type="hand" corresp="#h1"/>
+                                    (<bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">47</citedRange></bibl>).</desc>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="1v" to="4"/>
+                                    <locus target="#172v"/>
+                                    <desc type="GuestText"><title type="complete" ref="LIT7328MalkeaYostos"/>
+                                    </desc>
+                                </item>
+                            </list>
+                            <list>
+                                <item xml:id="a2">
+                                    <desc type="Unclear">
+                                        <bibl>Without specifying on which folio,
```

Comment:
I think the element bibl should surround only the reference proper

Reply:
You are right!